net/Socket.cpp

```diff
@@ -690,6 +693,7 @@ std::shared_ptr<Socket> ServerSocket::accept()
 
 int Socket::getPid() const
 {
+#ifndef __FreeBSD__
```

Comment:
`SOL_SOCKET SO_PEERCRED` isn't standard either, so this should be specific to Linux. (And OpenBSD IIRC?)